README.md

```diff
@@ -53,9 +55,10 @@
 </details>
 
 <!-- ABOUT THE PROJECT -->
+
 ## Intro
 
-The [COVESA Vehicle Signal Specification](https://covesa.github.io/vehicle_signal_specification/) (VSS) defines the names and semantics of a large set of *data entries* that represent the current and/or intended state of a vehicle's sensors and actuators organized in a tree-like structure. For example, the vehicle's current speed is represented by the `Vehicle.Speed` entry.
+The [COVESA Vehicle Signal Specification](https://covesa.github.io/vehicle_signal_specification/) (VSS) defines the names and semantics of a large set of _data entries_ that represent the current and/or intended state of a vehicle's sensors and actuators organized in a tree-like structure. For example, the vehicle's current speed is represented by the `Vehicle.Speed` entry.
```

Comment:
Is there any particular reason why you prefer `_something_` (_something_) for italics rather than `*something*` (*something*). They both result in the same result and are listed at https://daringfireball.net/projects/markdown/syntax , but maybe there is a style guide somewhere that prefers one over the other?

Reply:
The formatting comes from my default formatter prettier:
https://spec.commonmark.org/0.31.2/

as you stated, both are valid and both are resolving to <p><em>foo bar</em></p>.
I can change it back to the initial spelling, no problem :)

Reply:
Nah, I was just curious.

And does https://spec.commonmark.org/0.31.2/ actually state that underscore is preferred over asterisk? I found a lot of examples of both in section 6.2, but no clear statements that one is preferred over the other I think.

Reply:
I actually also didnt find if there are any preferations :D